vector/src/debug/java/im/vector/app/features/debug/features/DebugVectorFeatures.kt

```diff
@@ -116,11 +119,12 @@ private fun <T : Enum<T>> enumPreferencesKey(type: KClass<T>) = stringPreference
 object DebugFeatureKeys {
     val onboardingAlreadyHaveAnAccount = booleanPreferencesKey("onboarding-already-have-an-account")
     val onboardingSplashCarousel = booleanPreferencesKey("onboarding-splash-carousel")
-    val onboardingUseCase = booleanPreferencesKey("onboarding-splash-carousel")
+    val onboardingUseCase = booleanPreferencesKey("onboarding-use-case")
```

Comment:
Strange to see this change in this PR.

Reply:
I observed that the key was duplicated so I changed it in a commit, I agree this is not related to this PR but not sure it will make sense to revert the commit? 🤔

Reply:
This is definitely an issue, but maybe (next time) create a quick dedicated PR which can me merged super fast (at least faster than this one). You can even create it from GitHub if you do not want to switch branch locally.

Reply:
(done in #6456)